### PR TITLE
adds test4

### PR DESCRIPTION
**Pull request by andyloose**
Validates that all section of the dashboard are present

### CODE REVIEW

File: tests/test_example_assert_dashboard_page.py [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+        driver.get("https://opensource-demo.orangehrmlive.com/web/index.php/auth/login")
+        time.sleep(2)
+        login_hrm_page = LoginHrmPage(driver)
+        login_hrm_page.type_username("Admin")
+        login_hrm_page.type_password("admin123")
+        login_hrm_page.click_button_login()
+        time.sleep(5)
+
+        dashboard_page = DashboardPage(driver)
+        assert dashboard_page.get_title_time_at_work() == "Time at Work"
```
Comment (carleguizamon) [inline tests/test_example_assert_dashboard_page.py]: Please add all the error messages @andyloose

File: tests/test_example_assert_dashboard_page.py [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+        time.sleep(2)
+        assert dashboard_page.get_title_quick_launch() == "Quick Launch"
+        time.sleep(2)
+        assert dashboard_page.get_title_buzz_latest_posts() == "Buzz Latest Posts"
+        time.sleep(2)
+        assert dashboard_page.get_title_employees_on_leave_today() == "Employees on Leave Today"
+        time.sleep(2)
+        assert dashboard_page.get_title_employee_distribution_by_sub_unit() == "Employee Distribution by Sub Unit"
+        time.sleep(2)
+        assert dashboard_page.get_title_employee_distribution_by_location() == "Employee Distribution by Location"
```
Comment (carleguizamon) [inline tests/test_example_assert_dashboard_page.py]: Same here

File: tests/test_example_assert_dashboard_page.py [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+        time.sleep(2)
```
Comment (carleguizamon) [inline tests/test_example_assert_dashboard_page.py:33]: This last time.sleep is not needed

**Review by carleguizamon — CHANGES_REQUESTED** [review]

File: tests/test_example_assert_dashboard_page.py (hunk 1 above)
Comment (andyloose) [reply tests/test_example_assert_dashboard_page.py]: fixed all comments

File: tests/test_example_assert_dashboard_page.py (hunk 2 above)
Comment (andyloose) [reply tests/test_example_assert_dashboard_page.py]: fixed

File: tests/test_example_assert_dashboard_page.py (hunk 3 above)
Comment (andyloose) [reply tests/test_example_assert_dashboard_page.py:33]: fixed

**Pull request merged**